vscode/src/commands.ts

```diff
@@ -84,6 +84,23 @@ const commandsMap: (state: ExtensionState) => {
         console.error("Could not start the server", e);
       }
     },
+    "konveyor.getToken": async (): Promise<string | undefined> => {
+      let token = await state.extensionContext.secrets.get("konveyor.token");
+
+      if (!token) {
```

Comment:
Just a note, if they don't have auth enabled in the hub we won't need to use a token to connect